index.html

```diff
@@ -254,7 +254,18 @@ <h3>LET'S CONNECT</h3>
 					</div>
 				</li>
 				<hr>
-			</div>
+			</ul>
+			<div class="form-section">
+				<h1>Contact me</h1>
+				<p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+	I’d love to help with it!</p>
+				<form class="form" action="https://formspree.io/f/xnqybwkl" method="POST">
+					<input name="name" type="text" placeholder="Name" required>
```

Comment:
- [ ] It is required that the input should not be more than 30 and something similar but 300 for the text area. You can see the highlighted in the image below.
![image](https://user-images.githubusercontent.com/86200356/228338339-3782d757-b962-4015-b1ef-970cc9e7b4d1.png)